### PR TITLE
Align symbols horizontally in rofi's presentation

### PR DESCRIPTION
**Pull request by eepp**
See commit message for more details.

You should create a new screenshot (I don't have your exact setup) if you merge this feature.

By the way, I appreciate your work.

### CODE REVIEW

File: arthur-c.rasi [hunk 1]
```diff
@@ -63,7 +63,7 @@ configuration {
     highlight:     underline bold #ffffff;
 
     transparent: rgba(0,0,0,0);
-    font: "Ubuntu Mono 16";
```
Comment (NoraCodes) [inline arthur-c.rasi:66]: This change seems unrelated.

File: special_character_list [hunk 2]
```diff
@@ -1,2197 +1,2187 @@
-alpha   ga     :α
```
Comment (NoraCodes) [inline special_character_list:1]: Why is this entire file different?

**Review by NoraCodes — COMMENTED** [review]
I generally like the changes, thank you!

File: arthur-c.rasi (hunk 1 above)
Comment (eepp) [reply arthur-c.rasi:66]: Actually the new layout made some symbols invisible on the right for me, so I decreased the font size a bit to make them fit.

File: special_character_list (hunk 2 above)
Comment (eepp) [reply special_character_list:1]: Because the delimiter is tab now (see the commit message).

I also took the liberty to remove underscores at a few places, and putting alternatives between parentheses just for the look and feel.

File: arthur-c.rasi (hunk 1 above)
Comment (NoraCodes) [reply arthur-c.rasi:66]: Gotchya, understood.

File: special_character_list (hunk 2 above)
Comment (NoraCodes) [reply special_character_list:1]: Gotchya, seems alright. Thanks!

**Comment by NoraCodes** [comment]
There's a minor issue with alignments breaking but I'll merge this and just make that an issue.
